omod/src/main/webapp/pages/myaccount/changeDefaults.gsp

```diff
@@ -8,7 +8,40 @@
         primaryLocaleOptions.add([label: it.displayName, value: it.toString()])
     }
 %>
+<script type="text/javascript">
+    var value;
+  jq(document).ready(function() {
+  jq("select").on('change', function() {
+	  value = this.value;
```

Comment:
Shouldn't lines 15 and 16 be have the same indention?
Can you go through this whole fine again?